docs/email-signup.md

```diff
@@ -0,0 +1,15 @@
+# Email signup
+
+Currently only the business finder is making use of these features, but we believe other finders could use these in the future.
+
+## Subscribing to multiple facet values
+
+Users can [pick and choose which `facet_values` to subscribe to](https://github.com/alphagov/finder-frontend/pull/1191), and will be emailed according to their chosen frequency whenever content is newly tagged or existing tagged content is updated (provided the 'send email' checkbox is checked in content tagger).
+
+Now that subscriptions can be custom selected by the user, rather than curated by a content editor, we found that the subscription title became very long and difficult to read if selecting too many facet values. For this reason, we [added `EmailAlertListTitleBuilder` for creating comma-separated titles](https://github.com/alphagov/finder-frontend/pull/1164) that are easier to read. In the spirit of optimising code for deletion, much of the code is in separate files to the main title logic.
+
+email-alert-api treats `facet_values` as one 'group', so the 'OR logic' we want works out of the box. The finder itself used to rely on [passing `combine_mode: 'or'` to email-alert-api and defining an `OrJoinedFacetSubscriberList`](https://github.com/alphagov/email-alert-api/pull/867) when creating the subscription, but we're working to remove this.
+
+## Changing the default selected frequency
+
+We [added a `default_frequency` parameter](https://github.com/alphagov/finder-frontend/pull/1104) to finder-frontend and email-alert-frontend so that we could change the default selected frequency from "Immediately" to "Daily". This was in response to user research, where we found the quantity of EU Exit related updates was quite overwhelming.
```

Comment:
The 'why' part of this feels it should be in an ADR.  I'd recommend just explaining what's been added and how it can be used elsewhere.